dist/lightstep-tracer.js

```diff
@@ -19527,6 +19545,10 @@ var Tracer = function (_opentracing$Tracer) {
 
         _this._info('Tracer created with guid ' + _this._runtimeGUID);
 
+        if (_this._options.access_token.length === 0) {
+            _this._warn('No access token defined - this is only supported in developer mode.');
```

Comment:
I feel this message is overly specific. Ideally, the library would just print a message when failures happen, explaining that the reason was an invalid access token (with the token showing). I would be happy with an INFO message saying "Access token is not set -- this requires a satellite with access token checking disabled, e.g., a developer satellite". This leaves the customer an opportunity to ask for this feature, which customer-success could easily grant.

Reply:
(Satellite release notes describe the new flags, which are independent of dev-mode, so customer success is aware of these things and can give them to interested customers.)

Reply:
I've modified this language, ptal.